code/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Header = (props) => {
+    return (
+        <header>
+            <h1>{props.headerText}</h1>
+        </header>
+    );
+}
```

Comment:
The App component only has 2 spaces indentation - check your VSC settings so that tab size is set to 2 spaces and change so this goes for all your files in this project

Reply:
Fixed